### PR TITLE
Added twitter-stream-map api

### PR DESCRIPTION
**Pull request by barisesmer**
# Description

This is a simple app that makes use of twiter api via tweepy. Halil created the app and I added some new features to bring it to its latest state.

## Type of change

Please delete options that are not relevant.

- [x ] New feature (non-breaking change which adds functionality)

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Review by abdusco — APPROVED** [review]
Very well done. Would love to play with this once it's deployed.

File: twitter-api/twitter-stream-map/config.py [hunk 1]
```diff
@@ -0,0 +1,4 @@
+CONSUMER_KEY = 'jsjuaKAgWz4zDK2cKnGXf5Y30'
```
Comment (enisimsar) [inline twitter-api/twitter-stream-map/config.py:1]: Recommendation: Remove these lines and change your secrets from apps.twitter.com :)

File: twitter-api/twitter-stream-map/config.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
...
+CONSUMER_SECRET = 'vgAwEE77VIzVqvqVSS8yeSOMof3r38eF5vHiv6MhwQajk8sYHe'
```
Comment (enisimsar) [inline twitter-api/twitter-stream-map/config.py:2]: Recommendation: Remove these lines and change your secrets from apps.twitter.com :)

Comment (bayneri) [reply twitter-api/twitter-stream-map/config.py:2]: already did but thx @enisimsar

**Pull request merged**